setup.cfg

```diff
@@ -30,3 +74,35 @@ include_trailing_comma = true
 force_grid_wrap = 0
 skip = migrations
 not_skip = __init__.py
+
+[mypy]
+python_version = 3.9
```

Comment:
@hannseman Having the lowest supported version here helps us find compatibility issues. Having 3.9 here will make mypy silent about some potential issues it could report (like importing from typing instead of typing-extensions).

Reply:
Ah, makes sense